src/main/java/frc/robot/commands/Climbernetwork.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.commands;
+
+import com.ctre.phoenix.motorcontrol.can.MotControllerJNI;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class Climbernetwork extends CommandBase {
```

Comment:
```suggestion
public class TestClimber extends CommandBase {
```

This should eventually be able to run the climber in the simpliest way possible, if it becomes to complex to run just over network tables, then a series of commands should be used, or even switching to "production" quality commands.